site/architecture.md

```diff
@@ -5,7 +5,7 @@ The operator consists of the following two main parts:
 *	The operator itself, a process that runs in a Docker container deployed into a Kubernetes pod and which performs the actual management tasks.
 *	A Kubernetes job which can be used to create a WebLogic domain.
 
-The operator is packaged in a Docker image `container-registry.oracle.com/middleware/weblogic-operator:latest`.  This image can be deployed to a Kubernetes cluster.  It is recommended that the operator be deployed in its own namespace.  Only one operator is permitted in a namespace; however, multiple operators may be deployed in a Kubernetes cluster provided they are each in their own namespace and the list of namespaces they manage do not overlap.
+The operator is packaged in a Docker image `https://hub.docker.com/r/oracle/weblogic-kubernetes-operator/`.  This image can be deployed to a Kubernetes cluster.  It is recommended that the operator be deployed in its own namespace.  Only one operator is permitted in a namespace; however, multiple operators may be deployed in a Kubernetes cluster provided they are each in their own namespace and the list of namespaces they manage do not overlap.
```

Comment:
@rosemarymarano, You've changed the sense of the information.  Before, it gave the docker image name and tag you would use in a `docker pull` command.  Now, you've changed it to the web page that provides that information.  If it were up to me :), I'd have the given link for the text, "Docker image" and then have a section that shows the correct pull command, which is `docker pull docker pull oracle/weblogic-kubernetes-operator:1.1`.  @mriccell, what do you think?

Reply:
I agree. I'll change it.